sources/Inputs/src/Node.cpp

```diff
@@ -67,5 +68,61 @@ operator>=(const Node& lhs, const Node& rhs) {
 
 VoltageLevel::VoltageLevel(const VoltageLevelId& vlid) : id(vlid) {}
 
+/////////////////////////////////////////////////
+
+std::shared_ptr<Line>
+Line::build(const LineId& lineId, const std::shared_ptr<Node>& node1, const std::shared_ptr<Node>& node2) {
+  auto ret = std::shared_ptr<Line>(new Line(lineId, node1, node2));
+  node1->neighbours.push_back(node2);
```

Comment:
What if node1 or node2 is nullptr?

Reply:
The line is not created because we check that outside the builder

Reply:
could you please add an assert ?

Reply:
OK